src/webhooks/pubsub/slackScores.ts

```diff
@@ -247,99 +243,76 @@ export const sendGitHubActivityMetrics = async (
+
+  const createTable = (
+    items: discussionInfo[] | commenterInfo[] | issueInfo[],
+    firstColumnName,
     secondColumnName,
-    'numComments'
-  )}|\n├${'─'.repeat(
-    DISCUSSION_COLUMN_WIDTH + NUM_COMMENTS_COLUMN_WIDTH + NUM_ROW_SPACES
-  )}┤\n`;
-  githubCommenters
-    .slice(0, NUM_DISCUSSION_SCOREBOARD_ELEMENTS)
-    .forEach((discussionCommenterInfo: discussionCommenterInfo) => {
-      const usernameText = addSpaces(
-        discussionCommenterInfo.username,
-        'firstColumnItem'
-      );
-      const numCommentsText = addSpaces(
-        discussionCommenterInfo.num_comments.toString(),
-        'numComments'
-      );
-      scoreBoard += `| ${usernameText}| ${numCommentsText}|\n`;
+    type
+  ) => {
+    let scoreBoard = `\n┌${'─'.repeat(
+      DISCUSSION_COLUMN_WIDTH + NUM_COMMENTS_COLUMN_WIDTH + NUM_ROW_SPACES
+    )}┐\n| ${addSpaces(firstColumnName, 'firstColumnItem')}│ ${addSpaces(
+      secondColumnName,
+      'numComments'
+    )}|\n├${'─'.repeat(
+      DISCUSSION_COLUMN_WIDTH + NUM_COMMENTS_COLUMN_WIDTH + NUM_ROW_SPACES
+    )}┤\n`;
+    items.slice(0, NUM_DISCUSSION_SCOREBOARD_ELEMENTS).forEach((itemInfo) => {
+      if (type === 'commenters') {
+        const usernameText = addSpaces(itemInfo.username, 'firstColumnItem');
+        const numCommentsText = addSpaces(
+          itemInfo.num_comments.toString(),
+          'numComments'
+        );
+        scoreBoard += `| ${usernameText}| ${numCommentsText}|\n`;
+      } else {
+        // Append less than sign here, because trailing spaces are ignored for text within the <> blocks for a hyperlink
+        const truncatedDiscussionTitle =
+          itemInfo.title.length <= DISCUSSION_COLUMN_WIDTH
+            ? itemInfo.title + '>'
+            : `${itemInfo.title.slice(0, DISCUSSION_COLUMN_WIDTH - 4)}...> `;
+        const truncatedDiscussionTitleWithSpaces = addSpaces(
+          // Remove all instances of ` char from string
+          truncatedDiscussionTitle.replace(/`/g, ''),
+          'discussion'
+        );
+        const link =
+          type === 'discussions'
+            ? `https://github.com/${itemInfo.repository}/discussions/${itemInfo.discussion_number}`
+            : `https://github.com/${itemInfo.repository}/issues/${itemInfo.issue_number}`;
+        const discussionText = `<${link}|${truncatedDiscussionTitleWithSpaces}`;
+        const numCommentsText = addSpaces(
+          itemInfo.num_comments.toString(),
+          'numComments'
+        );
+        scoreBoard += `| ${discussionText}| ${numCommentsText}|\n`;
+      }
     });
-  scoreBoard += `└${'─'.repeat(
-    DISCUSSION_COLUMN_WIDTH + NUM_COMMENTS_COLUMN_WIDTH + NUM_ROW_SPACES
-  )}┘`;
+    scoreBoard += `└${'─'.repeat(
+      DISCUSSION_COLUMN_WIDTH + NUM_COMMENTS_COLUMN_WIDTH + NUM_ROW_SPACES
+    )}┘`;
+    return scoreBoard;
+  };
+  const scoreBoard =
+    createTable(
+      discussions,
+      'Most Active Discussions this Week',
+      '# comments',
```

Comment:
nit: if this is the same param for all 3, can we get rid of it?

Reply:
yeah, will remove